contributors/design-proposals/raw-block-pv.md

```diff
@@ -521,18 +522,29 @@ type BlockUnmounter interface {
  	GetVolumePath() string
 }
 ```
-# Mounter binding matrix
+# Mounter binding matrix for statically provisioned volumes:
+
 | PV volumeType | Plugin fstype | PVC volumeType  | Result           |
 | --------------|:-------------:| ---------------:|-----------------:|
-|   unspecified | raw           | unspecified     | NO BIND          |
-|   unspecified | raw           | block           | BIND             |
-|   block       |               | unspecified     | NO BIND          |
-|   block       |               | block           | BIND             |
-|   block       |     ext4      | unspecified     | PLUGIN DEPENDENT |
-|   block       |     raw       | block           | BIND             |
-|   unspecified |     ext4      | block           | NO BIND          |
-|   unspecified |     raw       | block           | BIND             |
+|   unspecified |    --         | unspecified     | BIND             |
```

Comment:
Can we just use `file` instead of unspecified?  There should only be 4 rows in this table.